common/src/main/java/com/wynntils/wynn/model/map/poi/Label.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.model.map.poi;
+
+public class Label {
+    private String name;
+    private int x;
+    private int z;
+    private int layer;
+    private String level;
```

Comment:
I am not sure why level is a string? -1 for internal representation of undefined and return an optional in the getter? Or return -1?

Reply:
Because that is how it is in the original data. Most of the time it is an integer, but in some places it is like "30-40" or some other weird format. We could "fix" this manually and change these values, but currently the data is non-curated and only extracted from Wynncraft's official map, and converted to a sane json representation.